weave/trace_server/trace_server_interface.py

```diff
@@ -292,6 +292,20 @@ class TableCreateReq(BaseModel):
     table: TableSchemaForInsert
 
 
+class TableUpdateReq(BaseModel):
+    project_id: str
+    initial_digest: str
+    # insertions performed before popping so that it is easier for
+    # caller to know the index of insertion
+    insert_rows: typing.Optional[list[tuple[int, dict[str, typing.Any]]]] = None
+    pop_digests: typing.Optional[list[str]] = None
```

Comment:
are table rows guaranteed to be unique? what if the user intends to dedupe rows?